temperature/schemas.py

```diff
@@ -0,0 +1,13 @@
+import datetime
+
+from pydantic import BaseModel
+
+
+class Temperature(BaseModel):
+    id: int
+    city_name: str
```

Comment:
The field 'city_name' in the 'Temperature' schema should be 'city_id' to match the task description, which specifies that 'Temperature' should reference the city by 'city_id'.